ndb/src/google/cloud/ndb/_eventloop.py

```diff
@@ -345,25 +345,70 @@ def get_event_loop():
     raise exceptions.AsyncContextError()
 
 
-def add_idle(*args, **kwargs):
-    raise NotImplementedError
+def add_idle(callback, *args, **kwargs):
+    """Helper function.
 
+    Calls :method:`EventLoop.add_idle` on current event loop.
 
-def queue_call(*args, **kwargs):
-    raise NotImplementedError
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
```

Comment:
A Sphinx / napoleon tip: I'd just use `.AsyncContextError` here. It tells Sphinx to find the "nearest" relevant class named `AsyncContextError`.

Reply:
Ok